readme.md

```diff
@@ -13,9 +13,9 @@ Screenshot of bahn.guru:
 
 - improve shop link
 - optimize search time by passing parameters for departure & arrival time & max. changes
-- implement autocomplete with data from mav-stations
 - add additional discount cards (see mav-prices)
 - fix formatting for start & end time (add colon if missing)
+- train station autocomplete: cache requests to mav-stations API (e.g. for 1 or 30 days)
```

Comment:
Das verstehe ich nicht so ganz. Alle Stationsdaten liegen in einem lokal im Node Module "mav-stations" vor. Was möchtest du da cachen?

Reply:
Okay. Wenn das schon statisch ist, dann ist es ja schon gecacht. Das wusste ich nicht.